### PR TITLE
New traitor item- Hackimplant

### PR DESCRIPTION
**Pull request by tigercat2000**
This PR adds a new traitor implant labelled simply "Hackimplant".
* Allows user to control doors like the cyborg/AI, at limit of a cooldown between button presses
* Allows user to remotely access doors within view range
* Costs 20TC currently, can be changed.
* 10 tick cooldown, modifiable with one variable change
* Shining a penlight in an implanted person's eyes gives a special message.

### CODE REVIEW

**Comment by SamCroswell** [comment]
Personally I don't think this is a good idea -- Emags already serve this purpose.

**Comment by tigercat2000** [comment]
This serves as a higher-cost stealthy method of door access- Emag's leave a very visible signature, and cannot get through bolted doors. This, however, allows you to use the full functions of the door that borgs/AI's can, including unbolting, electrifying, and remote control.

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -753,6 +757,12 @@ About the new airlock wires panel:
 				emergency = 1
 				usr << "Emergency access has been enabled."
 
+	if(ishuman(usr))
+		if(isdhacker(usr) || ishacker(usr))
+			for(var/obj/item/weapon/implant/hack/H in usr)
+				H.powerlevel = 0
+				usr << "\red <b>Your hack-implant discharges it's energy to hack the door!</b> You will need to wait a minute for it to recharge."
```
Comment (Markolie) [inline code/game/machinery/doors/airlock.dm]: Should be its.

**Comment by Regen1** [comment]
That's like saying we should only have one laser weapon and one ballistic weapon, this is nowhere NEAR emag, for a lot of reasons @SamCroswell 
1: It is a lot easier to conceal
2: It offers more stealth
3: It leaves no(?) trace and does not show who actually accessed the door, so it's an easy way to get the AI killed.
4: It can not be found on your person unless you are searched with a penlight
5: Emags has a lot more uses on other objects other than doors.

**Comment by SamCroswell** [comment]
No, I'm saying we should have only one item to fill one niche. For instance: The revolver and stetchkin break this rule by both being loud-style traitor items used at a range with a clip size within one bullet of eachother. An item that leaves no trace of having been used and can only be checked by shining a penlight is someones eye is only going to encourage the valid-hunting antag chompers that we're trying to STOP security from becoming.

**Comment by Fox-McCloud** [comment]
:-1:

This is what emags and regular hacking are for, not to mention are handled well within the code themselves without the addition of any further snowflake; people are more than capable of doing this with remote signalers and engineering know-how---and all of this is accomplished without further snowflake and is already accessible.
 
Big no from me.

**Comment by Spacemanspark** [comment]
I don't think this is a good idea, though it's creative.
Also, when the fuck are we removing infinite uses from emags.

**Comment by ZomgPonies** [comment]
No. Those kinds of implants would be better in a whole cybernetics framework, built up from scratch.

**Pull request closed without merging**